cloud-functions/background/main.py

```diff
@@ -3,6 +3,44 @@
 from googleapiclient import discovery
 from google.cloud import pubsub_v1
 
+# Imports the Google Cloud client library
+from google.cloud import bigquery
+
+def flowers_table_insert_rows(client, datarow):
+    SCHEMA = [
+    bigquery.SchemaField('KEY', 'INTEGER', mode='REQUIRED'),
+    bigquery.SchemaField('PREDICTION', 'INTEGER', mode='REQUIRED'),
+    bigquery.SchemaField('SCORE1','FLOAT', mode='REQUIRED'),
+    bigquery.SchemaField('SCORE2','FLOAT',mode='REQUIRED'),
+    bigquery.SchemaField('SCORE3','FLOAT',mode='REQUIRED'),
+    bigquery.SchemaField('SCORE4','FLOAT',mode='REQUIRED'),
+    bigquery.SchemaField('SCORE5','FLOAT',mode='REQUIRED'),
+    bigquery.SchemaField('SCORE6','FLOAT',mode='REQUIRED'),
+    ]
+
+    #my_bigquery.dataset(dataset_name).table(table_name).exists()  # returns boolean
+
+    """Insert / fetch table data."""
+    dataset_id = 'flowers_dataset'
+    table_id = 'flowers_table'
+    dataset = bigquery.Dataset(client.dataset(dataset_id))
+    try:
+     dataset = client.create_dataset(dataset)
+    except Exception as err:
+     print("dataset already exists, appending to the dataset")
+    dataset.location = 'US'
+    #to_delete.append(dataset)
+    table = bigquery.Table(dataset.table(table_id), schema=SCHEMA)
+    try:
+     table = client.create_table(table)
+    except Exception as err:
+     print("table already exists, appending to the table")
```

Comment:
Check the formatting here, that we are using tabs instead of spaces.
```
try:
  table = client.create_table(table)
except Exception as err:
  print("table already exists, appending to the table")
```